lib/config/migration.spec.ts

```diff
@@ -666,13 +667,14 @@ describe('config/migration', () => {
     expect(migratedConfig).toEqual({
       'pip-compile': {
         enabled: true,
-        fileMatch: [
-          '(^|/)requirements\\.txt$',
-          '(^|/)requirements-fmt\\.txt$',
-          '(^|/)requirements-lint\\.txt$',
-          '.github/workflows/requirements.txt',
-          '(^|/)debian_packages/private/third_party/requirements\\.txt$',
-          '(^|/).*?requirements.*?\\.txt$',
+        filePatterns: [
+          'requirements.txt',
```

Comment:
Why is this necessary?

Reply:
To make sure the `.in` is replaced with `.txt` for `filePatterns` too.